src/main/webapp/WEB-INF/web.xml

```diff
@@ -45,14 +45,6 @@
         <param-value>/WEB-INF/context/applicationContext.xml</param-value>
     </context-param>
     
-<!--     For some reason Tomcat 7.0.x wasn't resolving the taglibs in the tags folder automatically -->
-    <jsp-config>
-         <taglib>
-             <taglib-uri>http://my.wisc.edu/jsp/jstl/functions</taglib-uri>
-             <taglib-location>/WEB-INF/tags/uwfn.tld</taglib-location>
-         </taglib>
-     </jsp-config>
```

Comment:
Hmm... looks like my solution to the taglib resolution in Tomcat 7 wasn't the right one.  I'm glad you figured out a better solution.